x-pack/plugins/spaces/public/management/edit_space/edit_space_general_tab.tsx

```diff
@@ -245,15 +265,13 @@ export const EditSpaceSettingsTab: React.FC<Props> = ({ space, features, history
     );
   };
 
-  const validator = new SpaceValidator();
-
   return (
     <>
       {doShowAlteringActiveSpaceDialog()}
       {doShowConfirmDeleteSpaceDialog()}
 
       <CustomizeSpace
-        space={formValues}
+        space={getSpaceFromFormValues(formValues)}
```

Comment:
This ensures that default initials are used, if the space attributes doesn't have that field.